README.md

```diff
@@ -3,10 +3,34 @@ QuDynamics.jl
 
 [![Build Status](https://travis-ci.org/JuliaQuantum/QuDynamics.jl.png?branch=master)](https://travis-ci.org/JuliaQuantum/QuDynamics.jl)
 
-A JuliaQuantum package for solving dynamical equations in quantum mechanics.
+A [JuliaQuantum](http://juliaquantum.github.io/) package for solving Dynamical Equations in Quantum Mechanics.
 
-The package aims to provide a flexible and unified framework for solving dynamical equations in quantum mechanics. The package will make use of existing packages mainly QuBase.jl in addition to various other solver packages like ODE.jl, Expokit.jl.
-Some of the discussion regarding design and implementation can be found [here](https://github.com/JuliaQuantum/JuliaQuantum.github.io/issues/20) and [here](https://github.com/numfocus/gsoc/blob/7917b4fc08ce73ac74f8a2b9dd7929d994fc4282/2015/proposals/Amit.md). Updates of the implemented features can be followed on the [news page](http://juliaquantum.github.io/news/) of the JuliaQuantum website.
+Background
+----------
+The package is aimed at providing a unified framework for solvers for solving Dynamical Equations in Quantum Mechanics. Various solvers have been integrated from packages like [ODE.jl](https://github.com/JuliaLang/ODE.jl), [ExpmV.jl](https://github.com/marcusps/ExpmV.jl),
+[Expokit.jl](https://github.com/acroy/Expokit.jl), [QuDOS.jl](https://github.com/acroy/QuDOS.jl) using base features from [QuBase.jl](https://github.com/JuliaQuantum/QuBase.jl). QuBase.jl is the base package which includes most of the basic type constructs and methods used in Quantum Mechanics.
+
+Overview
+--------
+The following Dynamical Equations can be solved using this package :
+
+1. Schrodinger Equation
+2. Liouville von Neumann Equation
+3. Lindblad Master Equation
+
+Using the following solvers :
+
+1. Euler Method, Crank-Nicholson Method, Krylov subspace Method
+2. ode45, ode78, ode23s which have been integrated from [ODE.jl](https://github.com/JuliaLang/ODE.jl)
+3. Two versions of `expmv` implementations from :
+
+    a. [ExpmV.jl](https://github.com/marcusps/ExpmV.jl)
+    b. [Expokit.jl](https://github.com/acroy/Expokit.jl)
+4. Quantum Monte-Carlo Wave Function Method has been integrated with reference from [QuDOS.jl](https://github.com/acroy/QuDOS.jl)
```

Comment:
Also here you can drop the reference to QuDOS. I would rather refer to the paper by Dum et al (R. Dum, P. Zoller and H. Ritsch, Phys. Rev. A 45, 4879 (1992)).